mitype/keycheck.py

```diff
@@ -20,6 +20,27 @@ def is_escape(key):
         return ord(key) == curses.ascii.ESC
     return False
 
+def is_ignored(key):
+    keys = [
+        "KEY_UP",
+        "KEY_DOWN",
+        "KEY_LEFT",
+        "KEY_RIGHT",
+        "KEY_F0",
+        "KEY_F1",
+        "KEY_F2",
+        "KEY_F3",
+        "KEY_F4",
+        "KEY_F5",
+        "KEY_F6",
+        "KEY_F7",
+        "KEY_F8",
+        "KEY_F9",
+        "KEY_F10",
+        "KEY_F11",
+        "KEY_F12"
+    ]
+    return key in keys
```

Comment:
So this becomes something like:
```
def is_ignored(key):
keys = ["F1", "F2",......]
if sys.version_info[0] < 3:
        return key in keys
else:
        return isinstance(key,int))
```
So basically this function will return true if the key is to be ignored, else false.

Then at the end, inside the keycheck.is_valid_initial_key method, inside the if condition, add the following condition.
```
        or is_backspace(key)
        or is_ignored(key)
```